FlowCrypt/src/main/java/com/flowcrypt/email/database/dao/PubKeyDao.kt

```diff
@@ -41,4 +41,8 @@ interface PubKeyDao : BaseDao<PublicKeyEntity> {
 
   @Query("SELECT * FROM public_keys WHERE _id = :id")
   fun getPublicKeyByIdFlow(id: Long): Flow<PublicKeyEntity?>
+
+  companion object {
+    private const val SELECT_ALL_PUB_KEYS = "SELECT * FROM public_keys"
+  }
```

Comment:
That means in the future we may be doing this for all. But that's also for another issue.